client/payment-details/dispute-details/dispute-steps.tsx

```diff
@@ -0,0 +1,189 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { dateI18n } from '@wordpress/date';
+import moment from 'moment';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import type { Dispute } from 'wcpay/types/disputes';
+import { ChargeBillingDetails } from 'wcpay/types/charges';
+import { formatExplicitCurrency } from 'utils/currency';
+import { ClickTooltip } from 'wcpay/components/tooltip';
+
+interface Props {
+	dispute: Dispute;
+	customer: ChargeBillingDetails | null;
+	chargeCreated: number;
+	daysRemaining: number;
+}
+
+const DisputeSteps: React.FC< Props > = ( {
+	dispute,
+	customer,
+	chargeCreated,
+	daysRemaining,
+} ) => {
+	let emailLink;
+	if ( customer?.email ) {
+		const chargeDate = dateI18n(
+			'Y-m-d',
+			moment( chargeCreated * 1000 ).toISOString()
+		);
+		const disputeDate = dateI18n(
+			'Y-m-d',
+			moment( dispute.created * 1000 ).toISOString()
+		);
+		const emailSubject = `Problem with your purchase from ${ wcpaySettings.storeName } on ${ chargeDate }?`;
```

Comment:
I intentionally do not translate the email subject and body because it is a bit weird to translate a content that is for the customer who might not speak the same language as merchant.

Reply:
Can we keep it simple? This is site content, so it should be translatable. Just like content on pages, buttons, or in site emails. 

The merchant has control of all of this, but the default experience should not have mixed languages IMO.

Reply:
Note – this will not _send_ an email, just generate a draft email in merchant's email client.

Reply:
For example, if a multilingual merchant wants to send a localised "is everything ok with your purchase" email, they can edit in their mail client of choice or use another mailing solution such as MailPoet etc.

Reply:
I've made the email subject and body translatable in 787b8d9 to let the merchant decide what language to use.

Reply:
I think to default to the merchant's language is better than to suddenly see English if they have localized configuration.